### PR TITLE
Add a clipboard record tool 

### PR DESCRIPTION
**Pull request by lesterhnu**
This is the link to the project: [TITLE](URL)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [ ] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [ ] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Plugins/Integrations

- [ ] Works with **Tauri 1.x and onward**.
- [x] The project is open source and accepts contributions.
- [ ] The repo is at least 30 days old.
- [x] Documentation is in English.
- [x] The project is active and maintained.

### Templates

- [ ] Works with **Tauri 1.x and onward**.
- [ ] The repo is at least 30 days old.
- [ ] Documentation is in English.
- [ ] The template provides enough information about how to get started and what's included.
- [ ] The template is pretty different from the existing templates.

### Apps

- [x] The app is original and not too simple.
- [x] The README is in English.
- [x] The app makes a reasonable effort to be fast, lightweight and secure.
- [x] If the app closed source, evidence of it being built with Tauri is included.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,7 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 - [Padloc](https://github.com/padloc/padloc) - Modern, open source password manager for individuals and teams.
 - [Ghorbu Wallet](https://github.com/matthias-wright/ghorbu-wallet) - Cross-platform desktop HD wallet for Bitcoin.
 - [Stable Diffusion Buddy](https://github.com/breadthe/sd-buddy) - Desktop UI companion for the self-hosted Mac version of Stable Diffusion.
+- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content, using react-ts+sqlite+tauri
```
Comment (JonasKruckenberg) [inline README.md]: You should omit the references to technologies used, they might change down the road and offer no benefit to the reader.

```suggestion
- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content
```

But if you insist on having the packages in the description please wrap them in backticks.

```suggestion
- [Clipborad Record](https://github.com/lesterhnu/clipboard) - Record Clipboard Content, using `react-ts`+`sqlite`+`tauri`
```

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

**Comment by JonasKruckenberg** [comment]
Oh and you should change your apps icons from the default

**Comment by nothingismagick** [comment]
Also, maybe spell `clipboard` properly: `Clipborad Record` is a misspelling.

**Comment by lesterhnu** [comment]
> 
thanks for correction and I have updated the description and replaced icons from the default

**Comment by JonasKruckenberg** [comment]
Okay one last thing of note:

```json
"allowlist": {
      "all": true, // this will allow all APIs regardless of what comes next
      "window": {
        "all":true, // this does the same thing
        "show": true,
        "hide":true,
        "setSkipTaskbar":true
      },
      "notification":{
        "all": true
      }
    },
```

If you could just take a moment and fix up that allowlist, that'd be cool 😁

**Comment by lesterhnu** [comment]
I updated the allowlist, please check it

**Comment by JonasKruckenberg** [comment]
Thanks! 😁 If you're serious about security I would recommend setting a csp, currently yours is disabled (the "csp": null line). But I feel comfortable approving this PR as-is. Thank you for taking the time to respond to my feedback!

**Review by JonasKruckenberg — APPROVED** [review]

**Comment by JonasKruckenberg** [comment]
Oh and we require all commits to be signed across Tauri's repositories. Follow [this guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) to set up commit signing. Once all the commits in this PR are signed we can merge your contribution. Thanks for being part of our community!

**Comment by lesterhnu** [comment]
I wonder if this is ok;

**Comment by JonasKruckenberg** [comment]
Hmmm so we actually need *every* commit to be signed, I think you can follow [the command here](https://superuser.com/questions/397149/can-you-gpg-sign-old-commits) to sign every commit on this branch and then force push to replace them 🤔

**Comment by lesterhnu** [comment]
well this time it must be ok, I forget the `--force` when I push, I tried a lot of times and reset ...

**Comment by JonasKruckenberg** [comment]
> well this time it must be ok, I forget the `--force` when I push, I tried a lot of times and reset ...

Hahaha what an ordeal! But you did it 😁

**Pull request merged**